frontend/pages/dashboard/new-post.tsx

```diff
@@ -59,93 +59,91 @@ const NewPostPage: NextPage = () => {
 
   return (
     <AuthGate>
-      {(currentUser) => (
-        <DashboardLayout currentUser={currentUser}>
-          <form id="new-post">
-            <h1>Let's write a post</h1>
-
-            <label htmlFor="post-title">Title</label>
-            <input
-              className="j-field"
-              id="post-title"
-              value={title}
-              onChange={(e) => setTitle(e.target.value)}
-              type="text"
-              name="title"
-              placeholder="The Greatest Story Never Told..."
-              autoComplete="off"
-            />
-
-            <label htmlFor="post-language">Language</label>
-            <LanguageSelect
-              id="language"
-              languages={userLanguages}
-              value={langId}
-              onChange={setLangId}
-            />
-
-            <div className="editor-padding">
-              <JournalyEditor value={body} setValue={setBody} />
-            </div>
-
-            <div className="button-container">
-              <Button
-                type="submit"
-                disabled={!title || langId === -1}
-                variant={ButtonVariant.Primary}
-                data-test="post-submit"
-                onClick={(e: React.MouseEvent) => {
-                  e.preventDefault()
-                  createNewPost(PostStatusType.Published)
-                }}
-              >
-                Publish!
-              </Button>
-              <Button
-                type="submit"
-                disabled={!title || langId === -1}
-                variant={ButtonVariant.Secondary}
-                data-test="post-draft"
-                onClick={(e: React.MouseEvent) => {
-                  e.preventDefault()
-                  createNewPost(PostStatusType.Draft)
-                }}
-              >
-                Save Draft
-              </Button>
-            </div>
-            <style jsx>{`
+      <DashboardLayout>
```

Comment:
Only change in this file is deleting the `{(currentUser) => ()}` wrapper around `<DashboardLayout />`